examples/sw/tmd.sw

```diff
@@ -0,0 +1,143 @@
+# DATE: 2018-03-26 UNITS: metal CONTRIBUTOR: Jin-Wu Jiang jwjiang5918@hotmail.com
+# CITATION: J.-W. Jiang, Acta Mech. Solida. Sin 32, 17 (2019).
```

Comment:
Potential files in example folders should be symbolic links to their copy in the potentials folder to make certain they stay consistent.

Reply:
> @oywg11 thanks for your submission. However, this modification cannot be accepted in this form as it will silently give incorrect results for all accelerated versions of the "sw" pair style. My suggestion would be to follow the precedent of the "tersoff" potential, where there is a "tersoff/mod" pair style implemented as a derived class with modified versions of some of the functions used by the base PairTersoff class. The name of that new pair style could be something like "sw/angle" or "sw/mod". Some more detailed comments are added to individual changes.

@akohlmey Following your suggestion, I modified this potential as a derived class of the original SW potential. However, I met an issue that needs your help. Since the modification is only in the three-body term, I kept it only in the derived class following the same structure as pair_tersoff_mod. The issue is that when I run simulations with sw/mod, I found it still uses the original three-body term instead of the modified one. Thus there must be something important that I didn't implement but I cannot find it. Could you give me some suggestions? Thanks!

Reply:
> 

I think I solved this issue.

Reply:
> 

I worked on this pull request in a virtual Linux machine and I failed to generate a link to the potential file. Could you fix this issue? Thanks!